english/modern-tech

```diff
@@ -0,0 +1,2 @@
+If you would like to learn more about today's modern tech please Click [here](http://phys.org/technology-news/software/)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.